src/generic/listctrl.cpp

```diff
@@ -2072,8 +2090,12 @@ void wxListMainWindow::OnPaint( wxPaintEvent &WXUNUSED(event) )
         int lineHeight = GetLineHeight();
 
         size_t visibleFrom, visibleTo;
+        size_t visibleEnd;
```

Comment:
Please initialize the variable together with its declaration (especially as it could be made `const` here).